stories/webapp/accountdashboard/WordPressIntegrationStory.js

```diff
@@ -0,0 +1,105 @@
+import React, { Fragment } from 'react';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import { withModalTrigger, Modal, Button, InputWithButton, Icon } from 'lib';
+
+const stories = storiesOf('Web app', module);
+
+const ModalTrigger = withModalTrigger({
+  children: <Icon name="integration" />,
+  types: ['icon-only']
+});
+
+stories.add('WordPressIntegrationStory', () => {
+  const modalSubHeader = (
```

Comment:
💚 I would stick this in a separate component when you get in the App, to keep this file readable and the component nice and composed.

In here I think this works fine though 🙏